lua/apg/cl_menu.lua

```diff
@@ -75,9 +75,9 @@ local function APGBuildToolHackPanel()
 	local panel = APG_panels[ "misc2" ]
 	panel.Paint = function( i, w, h ) end
 
-	utils.switch( panel, 0, 40, 395, 20, "Inject custom hooks into Fading Doors", "thFadingDoors" )
+	utils.switch( panel, 0, 40, 395, 20, "Inject custom hooks into Fading Doors", "fadingDoorHook" )
 	utils.switch( panel, 0, 75, 395, 20, "Activate FRZR9K (Sleepy Physics)", "sleepyPhys" )
-	utils.switch( panel, 0, 110, 395, 20, "Hook FRZR9K into collision (Experimental)", "hookSP" )
+	utils.switch( panel, 0, 110, 395, 20, "Hook FRZR9K into collision (Experimental)", "sleepyPhysHook" )

```

Comment:
Why not just go all out? `sleepyPhysicsHook` ?

Reply:
Same reason as "vehAntiGhost"